darts/timeseries.py

```diff
@@ -880,6 +880,39 @@ def is_within_range(self,
         index = self.time_index()
         return index[0] <= ts <= index[-1]
 
+
+    def map(self,
+            fn: Callable[[np.number], np.number],
+            cols: Optional[Union[List[int], int]] = None) -> 'TimeSeries':
+        """
+        Applies the function fn elementwise to all values in this TimeSeries, or, to only those
```

Comment:
One more small thing: In the docstring, when explicitly referring to the name of a function argument, we usually put ` ticks around the name of the argument. So in this case for `fn` and for `cols` this would be nice.
Once that is done, I think we can approve this PR.

Reply:
Done.